src/esp/assets/ResourceManager.cpp

```diff
@@ -1750,8 +1750,11 @@ void ResourceManager::initDefaultLightSetups() {
 }
 
 void ResourceManager::initDefaultMaterials() {
-  shaderManager_.set<gfx::MaterialData>(DEFAULT_MATERIAL_KEY,
-                                        new gfx::PhongMaterialData{});
+  auto defaultMaterial = new gfx::PhongMaterialData{};
+  defaultMaterial->ambientColor = Magnum::Color4{0.1};
+  defaultMaterial->diffuseColor = Magnum::Color4{0.7};
```

Comment:
Wouldn't it make more sense to assign reasonable defaults in the PhongMaterialData itself instead? Having a white ambient in there by default doesn't make sense either.

https://github.com/facebookresearch/habitat-sim/blob/d34c3452c52e88336f67b4811856adea5d5ac3b6/src/esp/gfx/MaterialData.h#L19-L30

That way one can rely on them to be set to correct values and only modify what's needed.

Reply:
Good point. Done. 👍